### ISSUE TITLE
Allow selection of font family

### ISSUE THREAD

**Issue by eshellman** [mgiraldo/tenprintcover-py#8]
Noto Sans was selected for glyph coverage rather than for beauty. It would be nice to add an option to select a preferred font family, perhaps in coordination with #6.

**Comment by jenstroeger** [comment]
Agreed. I was thinking to add an option `--fonts` to the command line which allows the user to specify a font folder and file pattern like so:
```bash
./tenprintcover.py --fonts /path/to/folder/NotoSans*.ttf
```
This can be implemented using a [glob](https://docs.python.org/3/library/glob.html). The `select_font(text)` function would then search and choose the correct font based on glyph coverage. If no font was found, [assert](https://docs.python.org/3.5/reference/simple_stmts.html#the-assert-statement).

**Comment by mgiraldo** [comment]
would this be useful here? https://github.com/google/python-fire

**Comment by jenstroeger** [comment]
Thanks, @mgiraldo. There are several debuggers for Python code (if logging doesn’t suffice), like [pudb](https://github.com/inducer/pudb) (my favorite), [atom](https://atom.io/packages/python-debugger), [PyCharm](https://www.jetbrains.com/pycharm/) (commercial), or the built-in [pdb](https://docs.python.org/3/library/pdb.html). I’m not quite sure how that one would be different?

**Comment by mgiraldo** [comment]
not a debugger but a command-line interface generator
